### ISSUE TITLE
Google Chrome Canary cask installed via Nix will not auto-update

### ISSUE THREAD

**Issue by victorhooi** [LnL7/nix-darwin#1030]
I have a macOS machine, with Nix, and Nix-darwin setup (using [these templates](https://github.com/dustinlyons/nixos-config/)).

I've just added the Google Chrome Canary cask to my system.

However, I've noticed that the app will not auto-update itself due to an error:

![CleanShot 2024-08-07 at 10 05 35@2x](https://github.com/user-attachments/assets/bea9ca12-3584-4512-b5b3-cdf676932f89)

From what I can tell - it seems to be a permissions error:

https://support.google.com/chrome/thread/253456872/chrome-update-failed-error-code-9-extra-code-0?hl=en

The Google Chrome Canary app seems to have been installed with the wrong permissions for the auto-updater:

```
 ls -lah /Applications | grep Chrome
drwxr-xr-x@  3 foobar  staff    96B May 28 04:22 Google Chrome Canary.app
drwxr-xr-x@  3 root       wheel    96B Jul 30 09:17 Google Chrome.app
bytedance@H36H4NQQKK /Applications>
```

The Google Chrome app was installed normally, whilst the Google Chrome Canary app was installed using Nix, via a Homebrew cask.

Do you know if it's possible to fix it, so that it copies it with the correct permissions please?

**Comment by bnjmnt4n** [comment]
It's unlikely that this has anything to do with Nix/nix-darwin, since it was installed via Homebrew. Perhaps you could try to remove/re-add the cask and uninstall/reinstall the application.

**Comment by malob** [comment]
Agreed with the above. This is a Homebrew cask / Chrome permissions issue, not something nix-darwin controls. I think this can be closed. cc @emilazy @Enzime
